src/hotspot/share/runtime/globals.hpp

```diff
@@ -2100,6 +2100,11 @@ const intx ObjectAlignmentInBytes = 8;
       "-XX:CRaCRestoreFrom and continue initialization if restore is "      \
       "unavailable")                                                        \
                                                                             \
+  product(ccstr, CRaCIgnoredFileDescriptors, NULL, "Comma-separated list "  \
```

Comment:
Should not `0,1,2` be the default option then? So it will be also possible to overwrite this default.

Reply:
I didn't use `0,1,2` as if you want to ignore FD 42 you would have to write `0,1,2,42` rather than just 42. Starting with standard FDs closed is extremely non-standard, and in that case we should do something about `System.out` as well to be consistent.

Reply:
Agree, the current implementation makes sense.